Tickets/dm 26245 (#2)
lsst-dm/lsstinstall: scripts/README.md

Newinstall.sh
=============

> This script is meant to be used for whom has no experience with conda/anaconda/miniconda.
> 
> It will provide a ready to use lsstinstall environment.
> 
> Use this script only if you don'y have an existing installation of
> anaconda/miniconda in your user/machine.

Reviewer: ```suggestion
This script is meant for users with no experience with conda/anaconda/miniconda.

It will provide a ready to use lsstinstall environment.

Use this script only if you don't have an existing installation of
anaconda/miniconda available.
```
